pom.xml

```diff
@@ -58,6 +58,7 @@
     <module>keyvault/azure-securtiy-keyvault-jca/run-with-command-line-client-side</module>
     <module>keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side</module>
     <module>keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-server-side</module>
+    <module>keyvault/spring-cloud-azure-starter-keyvault-secrets/key-vault-secret-client</module>
```

Comment:
I think you should also update the root readme and root pom

Reply:
Updated.